index.js

```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   rules: {
-    'accessing-ember-namespace': require('./lib/rules/accessing-ember-namespace')
+    'accessing-ember-namespace': require('./lib/rules/accessing-ember-namespace'),
+    'disallow-function-prototype-extension': require('./lib/rules/disallow-function-prototype-extension')
```

Comment:
Looking at [the list of rules](http://eslint.org/docs/rules/) on eslint's page, it seems like the rule of thumb is: start a rule with `no-` if it is used to disallow something. If the rule enforces something or is configurable to either require/disallow, use a descriptive term.
So maybe these could be called:
  * `no-function-prototype-extension`
  * `ember-namespace` or `destructure-ember-namespace` (configure whether to require destructuring Ember.X props, and configure which ones (optionally) to allow if destructuring is required) or `no-ember-global` (no accessing properties directly as Ember.X — although this seems onerous for a line like `export default Ember.X.extend(`).

bit of a :bike: shed..